includes/admin/class-amp-editor-blocks.php

```diff
@@ -136,10 +136,15 @@ public function enqueue_block_editor_assets() {
 			);
 		}
 
+		$script_deps_path    = AMP__DIR__ . '/assets/js/amp-editor-blocks.deps.json';
```

Comment:
Problem: these JSON files aren't getting created when doing `npm run build`

Reply:
Fixed in 9a9beca3089f20bf32eb72cadce65135d5edf2e8 and cc28abdf79b091aac3f7539f0f332d795a21363d

Reply:
Good. The list file changes is now:

```
A	wp-content/plugins/amp/assets/js/amp-block-editor.deps.json
A	wp-content/plugins/amp/assets/js/amp-block-validation.deps.json
A	wp-content/plugins/amp/assets/js/amp-editor-blocks.deps.json
A	wp-content/plugins/amp/assets/js/amp-post-meta-box.deps.json
A	wp-content/plugins/amp/assets/js/amp-stories.deps.json
A	wp-content/plugins/amp/assets/js/amp-validated-url-post-edit-screen.deps.json
A	wp-content/plugins/amp/assets/js/amp-validated-urls-index.deps.json
A	wp-content/plugins/amp/assets/js/amp-validation-detail-toggle.deps.json
A	wp-content/plugins/amp/assets/js/amp-validation-single-error-url-details.deps.json
A	wp-content/plugins/amp/assets/js/amp-validation-tooltips.deps.json
A	wp-content/plugins/amp/assets/js/wp-dom-ready.deps.json
A	wp-content/plugins/amp/assets/js/wp-i18n.deps.json
D	wp-content/plugins/amp/assets/js/amp-admin-pointer.js
M	wp-content/plugins/amp/amp.php
M	wp-content/plugins/amp/assets/css/amp-stories-compiled-rtl.css
M	wp-content/plugins/amp/assets/css/amp-stories-compiled.css
M	wp-content/plugins/amp/assets/js/amp-block-editor.js
M	wp-content/plugins/amp/assets/js/amp-block-validation.js
M	wp-content/plugins/amp/assets/js/amp-editor-blocks.js
M	wp-content/plugins/amp/assets/js/amp-post-meta-box.js
M	wp-content/plugins/amp/assets/js/amp-stories.js
M	wp-content/plugins/amp/assets/js/amp-validation-tooltips.js
M	wp-content/plugins/amp/assets/js/wp-dom-ready.js
M	wp-content/plugins/amp/assets/js/wp-i18n.js
M	wp-content/plugins/amp/includes/admin/class-amp-editor-blocks.php
M	wp-content/plugins/amp/includes/admin/class-amp-post-meta-box.php
M	wp-content/plugins/amp/includes/class-amp-story-post-type.php
M	wp-content/plugins/amp/includes/sanitizers/class-amp-style-sanitizer.php
M	wp-content/plugins/amp/includes/validation/class-amp-validated-url-post-type.php
M	wp-content/plugins/amp/includes/validation/class-amp-validation-error-taxonomy.php
M	wp-content/plugins/amp/includes/validation/class-amp-validation-manager.php
```